src/reducer/admin/notification.js

```diff
@@ -3,12 +3,12 @@ import {
     HIDE_NOTIFICATION,
 } from '../../actions/notificationActions';
 
-const defaultState = {
+export const initialState = {
```

Comment:
I agree, `initialState` is the most commonly used. Would you mind using it everywhere ?